### PR TITLE
i18n(fr): Update install guides

### PR DESCRIPTION
**Pull request by agriffard**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate https://docs.astro.build/fr/install/auto/ with the same structure than https://docs.astro.build/en/install/auto/
- Translate https://docs.astro.build/fr/install/manual/ with the same structure than https://docs.astro.build/en/install/manual/
- Fix markdown in en/install/auto and en/install/manual.

### CODE REVIEW

**Review by delucis — CHANGES_REQUESTED** [review]
Awesome, thanks @agriffard! Let’s ask on Discord if anyone’s up for reviewing this and the other PRs.

In the meantime, could you remove the changes to the English pages included here? I think they’re probably correct, but they would mark other languages as out-of-date in #438, so best to avoid mixing languages in one PR.

**Comment by agriffard** [comment]
> could you remove the changes to the English pages included here?

Ok, changes to `en` pages reverted

File: src/pages/fr/install/manual.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
 title: Installer Astro manuellement
 description: Comment installer Astro manuellement avec NPM, PNPM, ou Yarn.
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:3]: Cute little addition
```suggestion
description: Comment installer Astro manuellement avec NPM, PNPM, ou Yarn avec l'outil en ligne de commande create-astro
```

**Comment by MoustaphaDev** [comment]
@agriffard I'm sorry I was reading the wrong file, it was correct 🤦

File: src/pages/fr/install/manual.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
 ---
 title: Installer Astro manuellement
-description: Comment installer Astro manuellement avec NPM, PNPM, ou Yarn.
+description: Comment installer Astro manuellement avec NPM, PNPM, ou Yarn avec l'outil en ligne de commande create-astro.
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:3]: revert
```suggestion
description: Comment installer Astro manuellement avec NPM, PNPM, ou Yarn.
```
Sorry again.

File: src/pages/fr/install/manual.md [hunk 3]
```diff
@@ -115,7 +145,26 @@ Si vous souhaitez inclure un [composant de Framework](/fr/core-concepts/framewor
 
 📚 Lisez l'[API de référence](/fr/reference/configuration-reference/) d'Astro pour plus d'informations.
 
-## 6. Étapes Suivantes
+## 6. Create `tsconfig.json`
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:148]:
```suggestion
Créer un fichier \`tsconfig.json\`
```

File: src/pages/fr/install/manual.md [hunk 4]
```diff
@@ -115,7 +145,26 @@ Si vous souhaitez inclure un [composant de Framework](/fr/core-concepts/framewor
...
+
+Typescript est configuré en utilisant `tsconfig.json`. Même si vous n'écrivez pas de code TypeScript, ce fichier est important afin que les outils comme Astro et VS Code sachent de quelle façon comprendre votre projet. Certianes fonctionnalités (comme les imports de packages npm) ne sont pas complètement supportés dans l'éditeur sans un fichier `tsconfig.json`.
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:150]: Typo
```suggestion
Typescript est configuré en utilisant `tsconfig.json`. Même si vous n'écrivez pas de code TypeScript, ce fichier est important afin que les outils comme Astro et VS Code sachent de quelle façon comprendre votre projet. Certaines fonctionnalités (comme les imports de packages npm) ne sont pas complètement supportés dans l'éditeur sans un fichier `tsconfig.json`.
```

File: src/pages/fr/install/manual.md [hunk 5]
```diff
@@ -115,7 +145,26 @@ Si vous souhaitez inclure un [composant de Framework](/fr/core-concepts/framewor
...
+
+Si ovus avez l'intention d'écrire du code TypeScript, l'utilisation du template Astro `strict` or `strictest` est recommendé. Vous pouvez voir et comparer les trois configurations de template à [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:152]: Typo
```suggestion
Si vous avez l'intention d'écrire du code TypeScript, l'utilisation du template Astro `strict` or `strictest` est recommendé. Vous pouvez voir et comparer les trois configurations de template à [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```

File: src/pages/fr/install/manual.md [hunk 6]
```diff
@@ -115,7 +145,26 @@ Si vous souhaitez inclure un [composant de Framework](/fr/core-concepts/framewor
...
+
+Créer `tsconfig.json` à la racine de votre projet, et copiez le code ci-dessous à l'intérieur. (Vous pouvez utiliser `base`, `strict` ou `strictest` pour votre template TypeScript) :
```
Comment (MoustaphaDev) [inline src/pages/fr/install/manual.md:154]: This fits better in the sentence IMO 
```suggestion
Créez `tsconfig.json` à la racine de votre projet, et copiez le code ci-dessous à l'intérieur. (Vous pouvez utiliser `base`, `strict` ou `strictest` pour votre template TypeScript) :
```

File: src/pages/fr/install/auto.md [hunk 7]
```diff
@@ -2,150 +2,100 @@
...
-
-# PNPM
-pnpm install
-```
-
-## 3. Lancer Astro ✨
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # create a new project with npm
```
Comment (MoustaphaDev) [inline src/pages/fr/install/auto.md:37]: We can translate comments too
```suggestion
# créer un nouveau projet avec npm
```

File: src/pages/fr/install/auto.md [hunk 8]
```diff
@@ -2,150 +2,100 @@
...
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # create a new project with pnpm
```
Comment (MoustaphaDev) [inline src/pages/fr/install/auto.md:43]:
```suggestion
  # créer un nouveau projet avec pnpm
```

File: src/pages/fr/install/auto.md [hunk 9]
```diff
@@ -2,150 +2,100 @@
...
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # create a new project with yarn
```
Comment (MoustaphaDev) [inline src/pages/fr/install/auto.md:49]:
```suggestion
  # créer un nouveau projet avec yarn
```

File: src/pages/fr/install/auto.md [hunk 10]
```diff
@@ -2,150 +2,100 @@
...
-yarn build
-
-# PNPM
-pnpm run build
-```
-
-Quand la commande se termine, vous devriez avoir un nouveau répertoire `dist/` dans votre projet que vous pouvez déployer directement sur votre hébergeur web favori.
-
-Pour commencer à déployer votre site web gratuitement, allez jeter un oeil à notre fier partenaire d'hébergement, [Netlify](https://www.netlify.com/). Pour obtenir des instructions sur la mise en place d'un déploiement, lisez notre [guide de déploiement](/fr/guides/deploy/).
+Si vous n'arrivez pas à ouvrir votre projet dans le navigateur, revenez au terminal où vous avez lancé le script `dev` pour voir ce qui a mal tourné,  ou si votre projet est affiché sur une url différente que celle mentionnée ci-dessus.
```
Comment (MoustaphaDev) [inline src/pages/fr/install/auto.md:89]: I might be nit-picking here but don't you think "servi" would be more accurate than "affiché" here?

Comment (agriffard) [reply src/pages/fr/install/auto.md]: ok, served => servi.

**Comment by MoustaphaDev** [comment]
Nice LGTM!

**Review by MoustaphaDev — APPROVED** [review]
Nice LGTM!

**Review by delucis — APPROVED** [review]
Thank you @agriffard! And thanks for reviewing @MoustaphaDev! 💜

**Pull request merged**